documentation/src/main/markdown/troubleshooting/solutions.md

```diff
@@ -4,28 +4,30 @@
 
 ### Symptom
 
-detect9.sh fails with: DETECT_SOURCE was not set or computed correctly, please check your configuration and environment.
+detect10.sh fails with: DETECT_SOURCE was not set or computed correctly, please check your configuration and environment.
 
 ### Possible cause
 
-detect9.sh is trying to execute this command:
+detect10.sh is trying to execute this command:
 ````
-curl --silent --header \"X-Result-Detail: info\" https://sig-repo.synopsys.com/api/storage/bds-integrations-release/com/synopsys/integration/synopsys-detect?properties=DETECT_LATEST
+curl --silent --header \"X-Result-Detail: info\" https://repo.blackduck.com/api/storage/bds-integrations-release/com/blackduck/integration/detect?properties=DETECT_LATEST_10
 ````
 The response to this command should be similar to the following:
 ```
 {
 "properties" : {
-"DETECT_LATEST" : [ "https://sig-repo.synopsys.com/bds-integrations-release/com/blackduck/integration/detect/10.0.0/detect-10.0.0.jar" ]
+"DETECT_LATEST" : [ "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/detect/10.0.0/detect-10.0.0.jar" ]
```

Comment:
@sig-cpotts One last change I promise! - This should be DETECT_LATEST_10 in the response